addons/mail/static/src/components/attachment/attachment.scss

```diff
@@ -50,7 +50,11 @@
 }
 
 .o_Attachment_details {
+    display: flex;
+    flex-flow: column;
+    justify-content: center;
     min-width: 0; /* This allows the text ellipsis in the flex element */
+    padding-inline-end : map-get($spacers, 1);
```

Comment:
If I understood right, add comment this is necessary to avoid the hover delete button from being too close to the text.

Reply:
PS: not sure where to comment it, and probably out of scope of current task, but I feel like the delete button is too big, and it's problematic that it comes only on hover.
The one that exists on attachment when posted looks better.